nebula-components/components/_icon.scss

```diff
@@ -0,0 +1,16 @@
+@if $nb-use-icon {
```

Comment:
Icon should _really_ be an SVG icon by default, the issue here with the background images is when a consumer changes the colour scheme the icons will not be coloured to match.
Also being SVG it allows the consumer to decide what icon they want there.